tests/issues/issue-317.php

```diff
@@ -4,14 +4,14 @@
     'issue-317-A' => [
         'ua' => 'Android-Cloud-X2/1.0 Linux/2.6.35.7 Android/2.3 Release/06.17.2013 Browser/AppleWebKit533.1 (KHTML, like Gecko) Mozilla/5.0 Mobile',
```

Comment:
This UA, like the one I added from #1630, doesn't really have the Android version anywhere in the UA, and is on Android 2.3, which we discussed some in #1630.  By adding the patterns I did to the Android generic file, this one got caught by that instead, which I think is probably more appropriate than Android 4.0.